plugin/helpers_bots.go

```diff
@@ -10,6 +10,13 @@ import (
 )
 
 func (p *HelpersImpl) EnsureBot(bot *model.Bot) (retBotId string, retErr error) {
+	const minimumSupportedVersion = "5.10.0"
```

Comment:
I don't see this re-used in any of the helper functions.  
Any reason for not going with a simple `p.ensureServerVersion("5.10.0")` instead? I think it reads nicely.